client/rest-high-level/src/test/java/org/opensearch/client/IngestClientIT.java

```diff
@@ -72,7 +76,11 @@ public void testGetPipeline() throws IOException {
         String id = "some_pipeline_id";
         XContentBuilder pipelineBuilder = buildRandomXContentPipeline();
         {
-            PutPipelineRequest request = new PutPipelineRequest(id, BytesReference.bytes(pipelineBuilder), pipelineBuilder.contentType());
+            PutPipelineRequest request = new PutPipelineRequest(
+                id,
+                BytesReference.bytes(pipelineBuilder),
+                (XContentType) pipelineBuilder.contentType()
```

Comment:
The type cast is not needed anymore, right? (MediaType is just fine)

Reply:
Correct. There were a few of those. I pushed a commit to remove all of those superfluous casts.